onnxruntime/python/onnxruntime_pybind_state.cc

```diff
@@ -1901,7 +1903,12 @@ RunOptions instance. The individual calls will exit gracefully and return an err
 
             return value;
           },
-          R"pbdoc(Get a single run configuration value using the given configuration key.)pbdoc");
+          R"pbdoc(Get a single run configuration value using the given configuration key.)pbdoc")
+      .def(
+          "set_adapter_active", [](RunOptions* options, lora::LoraAdapter* adapter) {
+            options->active_adapters.push_back(adapter);
+          },
+          R"pbdoc(Activates the specified lora adapter)pbdoc");
```

Comment:
naming suggestion on adding vs setting
```suggestion
          "add_active_adapter", [](RunOptions* options, lora::LoraAdapter* adapter) {
            options->active_adapters.push_back(adapter);
          },
          R"pbdoc(Adds the specified lora adapter as an active adapter)pbdoc");
```